.gitignore

```diff
@@ -13,6 +13,7 @@ dist
 dist-ssr
 *.local
 .env
+package-lock.json
```

Comment:
```suggestion
**/package-lock.json
```

Reply:
You also need to `git rm` the package-lock files since they have already been added to the repo.

Reply:
Seems to have removed them according to the diff. 
Is there a benefit to using the ** if the gitignore picks up those files without it?

Reply:
If the ignore file picks them up then ignore me.

Reply:
If the files are still in the repo after the merge you'll have to use the git rm command